site/en/fugu-showcase/fugu-showcase.njk

```diff
@@ -0,0 +1,245 @@
+---
+title: 'Project Fugu API Showcase'
+subhead: >
+  The Project Fugu API Showcase is a collection of apps that make use of APIs that were conceived in
+  the context of Project Fugu.
+description: >
+  The Project Fugu API Showcase is a collection of apps that make use of APIs that were conceived in
+  the context of Project Fugu.
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/8FZcBmFowbDKWxpkOytx.jpg
+alt: Blowfish swarm swimming in the ocean.
+---
+
+{% extends "layouts/base.njk" %}
+{% from 'macros/icon.njk' import icon with context %}
+
+{% block css %}
+{% InlineCss '/css/styleguide-main.css' %}
+{% endblock %}
+
+{% block title_bar %}
+  {% set sharingEnabled = true %}
+  {% include 'partials/breadcrumbs.njk' %}
+{% endblock %}
+
+{% block content %}
+  {% if hero %}
+    <div class="display-flex justify-content-center lg:pad-left-400 lg:pad-right-400">
+      {% Img
+        class="hero-image object-fit-cover",
+        src=hero,
+        alt=alt,
+        width="960",
+        height="480",
+        sizes="(min-width: 960px) 960px, 100vw"
+      %}
+    </div>
+  {% endif %}
+  <div class="display-flex gap-top-300 lg:gap-top-400">
+    <div class="display-flex justify-content-center width-full">
+      <article class="width-full pad-left-400 pad-right-400">
+        <div class="fugu-container width-full center-margin pad-400">
+          <form class="search-apps-apis pad-top-400">
+            <div class="display-flex align-items-center">
+              <p class="search-input">
+                {{ icon('search', {hidden: false}) }}
+                <input class="search-apps search-box__input" list="available-apps" placeholder="Search apps and APIs" aria-label="Search apps and APIs" />
+              </p>
+              <p>
+                <label for="example-enhanced-select">FILTER BY: </label>
```

Comment:
UI copy like the `placeholder` text and labels need to go through the `i18n` filter.